docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+# THIS IS TO BE USED FOR DEVELOPMENT AND TESTING PURPOSES ONLY
+version: '3.8'
+services:
+  synapse_release:
```

Comment:
Could you document the difference between `synapse_release`, `synapse_registration`, etc.?